wifiphisher/pywifiphisher.py

```diff
@@ -993,12 +873,12 @@ def run():
                     print term.move(1, term.width - 30) + "|" + " " + term.bold_blue("Wifiphisher " + VERSION)
                     print term.move(2, term.width - 30) + "|" + " ESSID: " + essid
                     print term.move(3, term.width - 30) + "|" + " Channel: " + channel
-                    print term.move(4, term.width - 30) + "|" + " AP interface: " + mon_iface.get_name() 
-                    print term.move(5, term.width - 30) + "|" + "_"*29 
-                    print term.move(1,0) + term.blue("Jamming devices: ")
-                    if os.path.isfile('/tmp/wifiphisher-jammer.tmp'):
-                        proc = check_output(['tail', '-5', '/tmp/wifiphisher-jammer.tmp'])
-                        print term.move(4,0) + proc 
+                    print term.move(4, term.width - 30) + "|" + " AP interface: " + mon_iface.get_name()
+                    print term.move(5, term.width - 30) + "|" + "_"*29
+                    print term.move(1, 0) + term.blue("Jamming the following clients: ")
+                    if deauthentication.get_clients():
+                        for client in deauthentication.get_clients():
+                            print client
```

Comment:
If you have more than 5 clients this will break to the next section (DHCP leases).